components/MapView.js

```diff
@@ -132,6 +132,14 @@ var MapView = React.createClass({
     pitchEnabled: PropTypes.bool,
 
     /**
+     * `Android only`
```

Comment:
nit: remove this line and add `@platform android` at the end of the prop description.